docs/general/Fees.md

```diff
@@ -10,9 +10,9 @@ every potential destination domain, with this mapping also outlining all potenti
 
 Fee strategy defines a set of rules on how fees should be charged when executing deposits on the source chain.
 
-### Static fee strategy
+### Fixed fee strategy
 
-This strategy always requires a predefined static fee amount per deposit. **It can only collect fees in the native
+This strategy always requires a predefined fixed fee amount per deposit. **It can only collect fees in the native
```

Comment:
`It can only collect fees in the native currency of the source chain` this is not true for substrate pallets, fee can be taken in token that is being transferred too when it's fungible

Reply:
You are right, let me fix this :+1: